src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -1,25 +1,17 @@
 import datetime
 import logging
 
-from sentry.issues.grouptype import ReplayDeadClickType
+from sentry.issues.grouptype import ReplayRageClickType
 from sentry.issues.issue_occurrence import IssueEvidence
 from sentry.replays.usecases.ingest.events import SentryEvent
 from sentry.replays.usecases.issue import new_issue_occurrence
 
 logger = logging.getLogger()
 
 
-def report_dead_click_issue(project_id: int, replay_id: str, event: SentryEvent) -> bool:
+def report_rage_click_issue(project_id: int, replay_id: str, event: SentryEvent):
```

Comment:
No return type here but mypy appears to be okay with it?  Guess it can infer `-> None`.

Reply:
yeah i think it does infer.